syntax = "proto2";
package aster.protos;

import "aster/protos/train.proto";
import "aster/protos/eval.proto";
import "aster/protos/input_reader.proto";
import "aster/protos/model.proto";

// Convenience message for configuring a training and eval pipeline. Allows all
// of the pipeline parameters to be configured from one file.
message TrainEvalPipelineConfig {
  optional Model model = 1;
  optional TrainConfig train_config = 2;
  repeated InputReader train_input_reader = 3;
  optional EvalConfig eval_config = 4;
  optional InputReader eval_input_reader = 5;
}
